tests/integration/test_requirements_dev.py

```
@@ -140,19 +153,25 @@
+        )
+
     def test_version_format_valid(self, requirements: List[Tuple[str, str]]):
         """Test that version specifications use valid format."""
-        version_pattern = re.compile(r'^(>=|==|<=|>|<|~=)\d+(\.\d+)*$')
-        
         for pkg, ver_spec in requirements:
             if ver_spec:
-                assert version_pattern.match(ver_spec)
+                try:
+                    SpecifierSet(ver_spec)
+                except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)